config/default.js

```diff
@@ -63,7 +62,8 @@ module.exports = {
 
   // The CDN host for AMO.
   amoCDN: addonsServerProdCDN,
-  staticHost: addonsFrontendCDN,
+  // addons-frontend statics are served by the CDN from the main domain.
+  staticHost: undefined,
```

Comment:
If this is undefined in the default config (and I don't see it defined anywhere else), could we remove the `staticHost` config parameter everywhere instead? It does not seem very useful.

Reply:
efe8423c7737e60c3bb5ff74ed7c2888eaf0fcd4

Reply:
@diox why not removing ALL occurrences of `staticHost`?

Reply:
mmm, that's a fair point, I suppose I could go all in and completely remove the preconnect to `staticHost` too, since it should always be `undefined`. I'll investigate this.

Reply:
https://github.com/mozilla/addons-frontend/pull/10893/commits/e0200ed40c1a8f94eff82c9a98404ae68c8f3784